src/components/search-filters/desktop/filters.tsx

```diff
@@ -0,0 +1,57 @@
+import { FiltersProps } from '../types';
+import FilterGroup from '../filter-group';
+
+const DesktopFilters: React.FunctionComponent<FiltersProps> = ({
+    className,
+    onFilter,
+    allFilters,
+    l1Items,
+    languageItems,
+    technologyItems,
+    contributedByItems,
+}) => {
+    return (
+        <div className={className}>
+            {!!l1Items.length && (
+                <FilterGroup
+                    title="Products"
+                    items={l1Items}
+                    filters={allFilters}
+                    setFilters={onFilter}
+                />
+            )}
+            {!!languageItems.length && (
+                <FilterGroup
+                    title="Language"
+                    items={languageItems}
+                    filters={allFilters}
+                    setFilters={onFilter}
+                />
+            )}
+            {!!technologyItems.length && (
+                <FilterGroup
+                    title="Technology"
+                    items={technologyItems}
+                    filters={allFilters}
+                    setFilters={onFilter}
+                />
+            )}
+            {/* <FilterGroup
```

Comment:
I assume we can delete this?

Reply:
Yeah but it may be implemented in the future so I think we can leave it for now.